integration_tests/commands/http/qwatch_test.go

```diff
@@ -0,0 +1,58 @@
+package http
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var qWatchQuery = "SELECT $key, $value WHERE $key LIKE \"match:100:*\" AND $value > 10 ORDER BY $value DESC LIMIT 3"
+
+func TestQWatch(t *testing.T) {
+	exec := NewHTTPCommandExecutor()
+
+	testCases := []TestCase{
```

Comment:
We should also be testing for updates to the result set being delivered over HTTP.

Reply:
Added integration test.